log/log.go

```diff
@@ -65,10 +65,9 @@ func New(log *syslog.Writer, stdoutLogLevel int, syslogLogLevel int) (Logger, er
 	}, nil
 }
 
-// initialize should only be used in unit tests.
+// initialize is used in unit tests and called by Get before the logger
+// is fully set up
```

Comment:
Small formatting nits.

```suggestion
// initialize is used in unit tests and called by `Get` before the logger
// is fully set up.
```